config/zones.json

```
@@ -4974,11 +4974,27 @@
     "timezone": null
   },
   "SA": {
+
+    "capacity": {
+      "biomass": 0,
+      "coal": 0,
+      "gas": 26753,
+      "hydro": 1000,
+      "nuclear": 0,
+      "oil": 26603,
+      "solar": 388,
+      "wind": 3,
+      "geothermal": 0
+    },
     "contributors": [
-      "https://github.com/q--"
+      "https://github.com/q--",
+      "https://github.com/Suwailem1"
+
     ],
     "parsers": {
-      "consumption": "GCCIA.fetch_consumption"
+      "consumption": "GCCIA.fetch_consumption",
+      "production": "GCCIA.fetch_production"
```

This should be `SA`, so it uses the new parser :)

```suggestion
      "production": "SA.fetch_production"
```

---

production and consumption, not just production correct?

---

Yeah, seems like the GCCIA parser isn't working anyway - so would make sense to use your's for both :)